atomos.framework/src/main/java/org/atomos/framework/base/AtomosRuntimeBase.java

```diff
@@ -108,6 +108,13 @@ public static AtomosRuntimeBase newAtomosRuntime() {
 		}
 	}
 
+	/** explicit call from AtomosRuntimBundle to allow resources to be freed.
```

Comment:
I'm not clear what will call this.  I think we need a close method on `ModuleConnectLoader` that would get called by `AtomosRuntimeModules.removeLayer` method